obs/preinstallimage-bios.sh

```diff
@@ -860,5 +859,13 @@ ln -srf /var/lib/bios/composite-metrics/agent_th  /var/lib/fty/fty-sensor-env/ag
 ln -srf /var/lib/bios/nut                /var/lib/fty/fty-nut
 ln -srf /var/lib/bios/uptime             /var/lib/fty/fty-kpi-power-uptime
 
+# make a state directory for fty agent
+mkftydir () {
+    mkdir "${1}" && chown bios:root "${1}" && chmod 0755 "${1}"
+}
+
+# Stop using tmpfiles.d et all - this repo is supposed to be system integration stuff
```

Comment:
Why? In practice you hardcode (or template-generate) paths that end up in the binaries or scripts via recipes in the component repos, which are outside of this repo scope. To change paths in one you must change the other. The stack of "ln"s above was a good or bad solution for the legacy compatibility, not for creation of the whole filesystem tree for all components whose existence you don't even know in advance. This is just bad systemic approach; tmpfiles were the better one (so OS image is set-up corresponding to actual requirements of each package as defined by its last build).